app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java

```diff
@@ -71,12 +81,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, rootView);
 
         mainUser = getArguments().getParcelable(HomePage.ARG_USER);
-
+        if (mainUser != null) {
```

Comment:
There should never be a situation where the user is null but Intelij was complaining about it so I added it. Since I know there isn't a case where the user would be null, is it fine to not include this?